clients/src/main/java/org/apache/kafka/common/network/PlaintextChannelBuilder.java

```diff
@@ -67,11 +71,13 @@ public void close() {}
         private final PlaintextTransportLayer transportLayer;
         private final KafkaPrincipalBuilder principalBuilder;
         private final ListenerName listenerName;
+        private final long sessionBeginTimeMs;
 
-        private PlaintextAuthenticator(Map<String, ?> configs, PlaintextTransportLayer transportLayer, ListenerName listenerName) {
+        private PlaintextAuthenticator(Map<String, ?> configs, PlaintextTransportLayer transportLayer, ListenerName listenerName, Time time) {
             this.transportLayer = transportLayer;
             this.principalBuilder = ChannelBuilders.createPrincipalBuilder(configs, transportLayer, this, null);
             this.listenerName = listenerName;
+            this.sessionBeginTimeMs = time.milliseconds();
```

Comment:
👍 We don't need this.  I added it in order to support the `long sessionBeginTimeMs()` method I had added to `Authenticator`, but now that the percentage calculation is moved into `SaslClientAuthenticator` we don't need the `sessionBeginTimeMs()` method on `Authenticator` anymore.   This file is now untouched.

Reply:
Oops, the file is not untouched because it still needs to send in a `Supplier<Authenticator>` rather than `Authenticator`  when it creates a `KafkaChannel` -- but the reference to `Time` is now gone.